### ISSUE TITLE
Add documentation warnings for experimental use cases

### ISSUE THREAD

**Issue by Flaxscrip** [KeychainMDIP/kc-docs#9]
It would be valuable to introduce a flexible documentation process to add visual callouts (like Warning, experimental, this will delete your DID, etc). 

I like the visuals proposed by @alexfornuto on Slack: 
![image](https://github.com/KeychainMDIP/kc-docs/assets/7094976/07de33b8-b618-4f22-864f-743f652a3232)
![image](https://github.com/KeychainMDIP/kc-docs/assets/7094976/729f1a53-d21d-4e0a-ba31-bf720b62379d)

**Comment by alexfornuto** [comment]
@Flaxscrip let me make sure I understand this issue.

We already have the ability to make these callouts ad-hoc. What I described out-of-band was an a way to quickly add repeatable callouts about things like experimental, alpha, and beta stage features to either the entire site or the top of a specific pages.

For site-wide notes, we can add a key/value combo to `docusaurus.config` setting the release stage of the project. I'm leaning towards page=-specific front-matter. While that would be more effort early on, it's more future-proof methodology when the project becomes stable and we introduce experimental/alpha/beta features ad-hoc.

Thoughts?

**Comment by Flaxscrip** [comment]
Thank you for clarifying. I like your callouts in the documentation. It allows us to excuse the low-security kc client; the calliuts help future wallet developers (like SelfID) to pay particular attention to the functionality. 

tldr: up to your judgement. It's OK to spend some time building functionality we will reuse.
